Waste.API/Models/WasteJourney.cs

```diff
@@ -28,5 +28,7 @@ public class WasteJourney : BaseEntity
         public int? WasteSubTypeId { get; set; }
 
         public virtual WasteSubType? WasteSubType { get; set; }
+
+        public string? WhatHaveYouDoneWaste { get; set; }
```

Comment:
I think the name should be simpler. We have 5 names for a database column here, which is a peculiar naming standard. Something shorter but descriptive will be more appropriate

Reply:
Done.